lib/relation.gi

```diff
@@ -1413,36 +1287,36 @@ InstallMethod(PrintObj, "for a binary relation on  n points", true,
 ##
 InstallGlobalFunction(EquivalenceRelationByPartition,
     function(X, subsX)
-	local fam, rel;
+        local fam, rel;
 
-	if not IsDomain(X) then
-		Error("Equivalence relations only constructible over domains");
-	fi;
+        if not IsDomain(X) then
+                Error("Equivalence relations only constructible over domains");
+        fi;
 
-	# make sure there are no repetitions
-	if not IsSet(AsSortedList(Concatenation(subsX))) then
-		Error("Input does not describe a partition");
-	fi;
+        # make sure there are no repetitions
+        if not IsSet(AsSortedList(Concatenation(subsX))) then
+                Error("Input does not describe a partition");
+        fi;
 
         #check that subsX is contained in X
-	if not  (IsSubset(X, AsSortedList(Concatenation(subsX)))) then
-		Error("Input does not describe a partition");
-	fi;
+        if not  (IsSubset(X, AsSortedList(Concatenation(subsX)))) then
+                Error("Input does not describe a partition");
```

Comment:
Weird indentation here, makes me wonder if this function should once again use less wide tabs... Except that there are a few bits and pieces (like the comment just above this place here) which are indented with spaces and fit with 8-wide tabs... But this might be an artifact of later edits.. Urgh

Reply:
Agreed it's weird, this file is one of the more inconsistent ones, I could re-indent the entire file, or just leave it?